go.mod

```diff
@@ -28,7 +28,7 @@ require (
 	github.com/pingcap/kvproto v0.0.0-20230726063044-73d6d7f3756b
 	github.com/pingcap/log v1.1.1-0.20221110025148-ca232912c9f3
 	github.com/pingcap/sysutil v0.0.0-20211208032423-041a72e5860d
-	github.com/pingcap/tidb-dashboard v0.0.0-20230911054332-22add1e00511
+	github.com/pingcap/tidb-dashboard v0.0.0-20230921035205-31ea86f64e20
```

Comment:
I noticed it's `release` brach, do we need to base on `release-6.5` branch?

Reply:
Nope, tidb-dashboard `release-6.5` branch is used for individual deployment, not for PD.

For PD, tidb-dashboard always use the internal version (version is the date format).